### PR TITLE
[Filebeat] Copy tag names from MISP data into event

### PR DESCRIPTION
**Pull request by hungnguyen-elastic**
## What does this PR do?

This PR adds the code to convert MISP tags to Elastic event Tags. This will open up automation opportunity in Elastic from MISP.

## Why is it important?

This will open up automation opportunity in Elastic from MISP. This also can allow threat content engineers to create threat matching or SIEM signal rules based on the MISP tags that are now added to the Elastic event through the MISP module

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] ~I have made corresponding changes to the documentation~
- [ ] ~I have made corresponding change to the default configuration files~
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

### CODE REVIEW

File: x-pack/filebeat/module/misp/threat/config/pipeline.js [hunk 1]
```diff
@@ -37,6 +37,18 @@ var threat = (function () {
         ignore_missing: true,
     });
 
+//Converting tags from MISP event to elastic event tags
+    var setTags = function (evt) {
+      var mispTags = evt.Get("json.Tag");
+      var elasticTags = [];
+      if (mispTags != null) {
+        mispTags.forEach(function(item){
+          elasticTags.push(item.name);
```
Comment (andrewkroh) [inline x-pack/filebeat/module/misp/threat/config/pipeline.js:46]: You can use `evt.AppendTo("tags", item.name)`.

File: x-pack/filebeat/module/misp/threat/config/pipeline.js [hunk 2]
```diff
@@ -37,6 +37,18 @@ var threat = (function () {
         ignore_missing: true,
     });
 
+//Converting tags from MISP event to elastic event tags
+    var setTags = function (evt) {
```
Comment (andrewkroh) [inline x-pack/filebeat/module/misp/threat/config/pipeline.js:41]:
```suggestion
    // Copy tags from MISP event to Elastic event tags.
    var setTags = function (evt) {
```

**Review by andrewkroh — CHANGES_REQUESTED** [review]
Thanks @hungnguyen-elastic 

- [x] Please add a JSON event to the test directory that contains this `Tag` field.
- [x] Add an entry to the CHANGELOG.next.asciidoc file under the Added/Filebeat section.
- [ ] Update the golden test files (`cd x-pack/filebeat; GENERATE=true PYTEST_ADDOPTS="-k misp" mage -v pythonIntegTest`). If you have trouble we can do this step.

**Review by andrewkroh — APPROVED** [review]

**Comment by hungnguyen-elastic** [comment]
Thanks @andrewkroh

**Pull request merged**